libs/blocks/icon-block/icon-block.css

```diff
@@ -240,6 +239,11 @@
   grid-template-columns: repeat(auto-fit, minmax(276px, 1fr)) !important;
 }
 
+.wrapper-anchor {
```

Comment:
This name is quite generic, it doesn't seem tied to the `.icon-block`, although it is. We should nest this to avoid future style conflicts.